### DISCUSSION TITLE
Terminal documentation really isn't clear

### DISCUSSION THREAD

**Discussion by MayNoii** [folke/snacks.nvim#39] [Q&A] [answered]
I see LazyVim just moved its terminal implementation here, but I'm really confused as to how to actually use it. I have a few questions:
1. How do I change the position of the opened window? Running `lua Snacks.terminal.open(nil, { win = { postion = "right" }})` on command mode still spawns the window on the bottom, on my end.
2. Is there any way to add borders to the floating window? I use full transparency with my theme, and the old term implementation (based on the lazy.nvim backend) allowed me to spawn a terminal with window borders. How would I do this here?
3. Is there any way to spawn a terminal in a new tab?

**Comment by folke** [comment]
you have a typo `postion` => `position`

**Comment by folke** [comment, marked as answer]
You can also globally configure this

```lua
  {
    "folke/snacks.nvim",
    opts = {
      terminal = {
        win = { position = "right" },
      },
    },
  },
```

**Reply by MayNoii** [reply]
Thank you! As for the borders, I also figured that out:
```lua
  {
    "folke/snacks.nvim",
    ---@type snacks.Config
    opts = {
      win = {
        border = "rounded",
      },
    },
  }
```
